### DISCUSSION TITLE
how to use quartznet ver3 ?

### DISCUSSION THREAD

**Discussion by keyone2693** [quartznet/quartznet#523] [General] [answered]
hi im trying  to work with ver3 with ver2.x doc but i have some errors
can u help me with that?

**Comment by lahma** [comment, marked as answer]
[Mailing list](https://groups.google.com/forum/#!forum/quartznet) is best for asking questions. Unfortunately it's hard to offer personal assistance with problems due to running the project is based on individual's free time.
